src/framework/session/core.clj

```diff
@@ -164,13 +164,19 @@
                                    :body   (json/write-value-as-string
                                              {:message "Invalid or missing session"})})))))
 
+(defn- check-excluded-resources
+  [protected-path excluded-resources uri]
+  (not (some (fn [resources]
+               (= uri (str protected-path resources)))
+             excluded-resources)))
+
 (defn- protect
   [protected-path
-   excluded-resource
+   excluded-resources
    {{uri :uri} :request
     :as        state}]
   (if (and (string/starts-with? uri protected-path)
-           (not= uri (str protected-path excluded-resource)))
+           (check-excluded-resources protected-path excluded-resources uri))
     (on-enter state)
```

Comment:
Shouldn't the check fn include the starts-with check?

Reply:
agreed, moved the code to do that